src/tqdm_publisher/publisher.py

```diff
@@ -21,7 +21,7 @@ def update(self, n=1, always_callback=False):
 
     # Subscribe to updates
     def subscribe(self, callback):
-        callback_id = uuid4()
+        callback_id = uuid4().hex
```

Comment:
Is there a reason you prefer the hexidecimal form (without dashes)? Just curious

Reply:
It doesn't really matter to me, aside from that it's a string rather than a UUID type. Do you have a preference?

Reply:
Oh, I forgot that we usually wrap `uuid4` in a string to get around that (`str(uuid4)`)

I'm personally more used to hashes / IDs having separators in them for whatever reason, but I might be biased. What have you seen more often?

Reply:
I associate separators with IDs that are truly unique for production applications. Otherwise, I just assume it's merely random

Reply:
I've updated to use `str(uuid4())` so we can maintain the original formatting.